modules/virt-about-node-maintenance.adoc

```diff
@@ -7,6 +7,11 @@
 
 Nodes can be placed into maintenance mode using the `oc adm` utility, or using `NodeMaintenance` custom resources (CRs).
 
+[NOTE]
+====
+As of version 4.11, `node-maintenance-mode` (NMO) is no longer shipped with {VirtProductName}. It is now deployed as a standalone operator with the Operator Lifecycle Manager (OLM).
```

Comment:
also worth to mentioned that NMO is not deployed _automatically_ by OLM or CNV. It is deployed by the admin using OperatorHub if s/he wishes to have it on his/her cluster.